views/snippets.mustache

```diff
@@ -0,0 +1,22 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>View Snippets</title>
+  </head>
+  <body>
+    <title>View Snippets</title>
+    <h1>Welcome {{username}}</h1>
+    <h1>These are your Code Snippets</h1>
+    {{#codeSnippets}}
+    <a href="/snippets/{{_id}}">{{title}}</a>
+    {{/codeSnippets}}
+    <hr>
+    <div class='links'>
+    <h4><a href='/logout'>Logout</a></h4>
+    <h4><a href="/create">Add your Snippet</a></h4>
+    <a href='/home'><h2>Cancel</h2></a>
+    <h4><a href='/home'>Go Home</a></h4>
+    <h4><a href='/'>Index</a></h4>
+    </div>
+  </body>
+</html>
```

Comment:
Let's add some indentation to this file so that it's easier to read. For instance, the content inside of your `<div>` (`div.links`) should be indented one level.

```html
<div class='links'>
  <h4><a href='/logout'>Logout</a></h4>
  <h4><a href="/create">Add your Snippet</a></h4>
  <a href='/home'><h2>Cancel</h2></a>
  <h4><a href='/home'>Go Home</a></h4>
  <h4><a href='/'>Index</a></h4>
</div>
```

You probably also want to indent any content inside of mustache tags (e.g. `{{#codeSnippets}} ... {{/codeSnippets}}`)

Reply:
I totally didn't catch that.  Thanks Ton!